liboqs/security.md

```diff
@@ -9,7 +9,7 @@ nav_order: 3
 
 ## Reporting security bugs
 
-If you think you have found a security bug in OQS software, please send email to security@openquantumsafe.org.  If you want to send an encrypted message, you can use [this PGP key](https://www.douglas.stebila.ca/about/contact/) to email dstebila@uwaterloo.ca.  We do not run a bug bounty program.
+If you think you have found a security bug in OQS software, please send email to security@openquantumsafe.org or [submit a security disclosure on Github](https://github.com/open-quantum-safe/liboqs/security).  If you want to send an encrypted message, you can use [this PGP key](https://www.douglas.stebila.ca/about/contact/) to email dstebila@uwaterloo.ca.  We do not run a bug bounty program.
```

Comment:
Link not applicable to oqsprovider

Reply:
Instead of including links to all of the projects what if we instead include a link to GitHub documentation: https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability ?